### PR TITLE
Add Kuzushiji-MNIST

### PR DESCRIPTION
**Pull request by abcdefg-1234567**

### CODE REVIEW

**Comment by abcdefg-1234567** [comment]
https://github.com/red-data-tools/red-datasets/issues/39

File: lib/datasets/kuzushiji-mnist.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  class KuzushijiMNIST < MNIST
+    BASE_URL = "http://codh.rois.ac.jp/kmnist/dataset/kmnist/"
+
+    private
+    def dataset_name
+      "Kuzushiji-MNIST"
+    end
+
+    def licenses
+      ["CC BY-SA 4.0"]
```
Comment (kou) [inline lib/datasets/kuzushiji-mnist.rb:13]: Could you use SPDX license ID https://spdx.org/licenses/ ?

```suggestion
      ["CC-BY-SA-4.0"]
```

**Review by kou — APPROVED** [review]
Thanks!

It looks almost good to me!

**Comment by kou** [comment]
Thanks!

**Pull request merged**